src/pages/about-us/tests/about-us.spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { ThemeProvider } from '@material-ui/styles';
+import { BrowserRouter } from 'react-router-dom';
+import { Button } from '@material-ui/core';
+import AboutUs from '../about-us';
+import { theme } from '../../../components/app/app-theme/app.theme';
+import TableContainerRow from '../../../containers/table-container-row';
+import TableContainerGenerator from '../../../containers/table-container-generator';
+
+jest.mock('react-redux');
+jest.mock('@apollo/client');
```

Comment:
We should use MockProvider for apollo tests 
https://github.com/ita-social-projects/horondi_admin/pull/1204/files#diff-50ce74b9bff850805606026b6e29904ac865e546831116fc1ea0d4cff181dc19